.github/workflows/build-electron.yml

```diff
@@ -22,12 +22,9 @@ on:
             - "master"
             - "feature/**"
             - "fix/**"
-        # paths:
-        #     - "packages/app/**"
-        #     - "packages/core/**"
-        #     - "packages/electron/**"
-        #     - "packages/pwa/**"
-        #     - "packages/locale/**"
+        paths:
+            - "packages/electron/**"
+            - "assets/**"
```

Comment:
Removing all packages except electron (PWA is loaded remotely so the other packages are not involved in this build) and adding the `assets` dir since that contains the app icon and name and other metadata used during the build process.

Reply:
Good point! I forgot that this only really loads the PWA, so it doesn't need to be rebuilt unless something in itself actually changes.